/*
 * This file was automatically generated by EvoSuite
 * Mon May 27 16:36:42 GMT 2024
 */

package projeto;

import org.junit.Test;
import static org.junit.Assert.*;
import org.evosuite.runtime.EvoRunner;
import org.evosuite.runtime.EvoRunnerParameters;
import org.junit.runner.RunWith;
import projeto.EncomendaExistenteException;

@RunWith(EvoRunner.class) @EvoRunnerParameters(mockJVMNonDeterminism = true, useVFS = true, useVNET = true, resetStaticState = true, separateClassLoader = true, useJEE = true) 
public class EncomendaExistenteException_ESTest extends EncomendaExistenteException_ESTest_scaffolding {

  @Test(timeout = 4000)
  public void test0()  throws Throwable  {
      EncomendaExistenteException encomendaExistenteException0 = new EncomendaExistenteException("2Zn");
  }
}
